src/parser/h6_parser.py

```
@@ -3,122 +3,142 @@
+    def parse(self, data: bytes) -> Optional[Union[Dict[str, str], List[Dict[str, str]]]]:
         """
-        H6レコードをパースしてフィールド辞書を返す
-
-        Args:
-            data: パース対象のバイトデータ
+        H6レコードをパースする。
 
-        Returns:
-            フィールド名をキーとした辞書、エラー時はNone
+        102,900バイトのフルストラクトの場合: List[Dict] を返す。
+        78バイトのフラットレコードの場合: 単一 Dict を返す（後方互換）。
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)